scylladb/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM scylladb/scylla
+
+# This downloads a prebuild static lib for hwloc 2.7.1 which includes a chromeos fix, and replaces the dynamically
+# linked version that scylla uses. See the README for more info
+
```

Comment:
Can we add a comment here to the Scylla issue tracker / our internal tracker here

Reply:
Nevermind, I see it below!